config/builds.js

```diff
@@ -37,7 +37,9 @@ const builds = buildConfigs
   })
   .map(buildConfig => {
     const name = buildConfig.name || '';
-    const env = buildConfig.env || {};
+    const env = buildConfig.env || {
+      SKU_TENANT: args.tenant || ''
```

Comment:
If I'm reading this right, this means that if you define environment variables, you'll no longer have access to the tenant option?